network-api/networkapi/wagtailpages/models.py

```diff
@@ -3,15 +3,18 @@
 )
 
 from .pagemodels.base import (
+    AreaOfFocus,
```

Comment:
`AreaOfFocus` and `FocusArea`.... so similar and so confusing 🙈. I'm gonna file a new ticket to rename them.

Reply:
Yes please. I originally had it called `AreaOfFocusOrderable` to explain what it is but that got renamed in another review.